tap_mysql/sync_strategies/common.py

```diff
@@ -74,7 +74,11 @@ def generate_select_sql(catalog_entry, columns):
         # if the column format is binary, fetch the values after removing any trailing
         # null bytes 0x00 and hexifying the column.
         if 'binary' == property_format:
-            escaped_columns.append(f'hex(trim(trailing CHAR(0x00) from {escaped_col})) as {escaped_col}')
+            escaped_columns.append(
+                f'hex(trim(trailing CHAR(0x00) from {escaped_col})) as {escaped_col}')
+        if 'spatial' == property_format:
```

Comment:
this needs to be `elif` and not `if`. Can you please change it, the tests are currently failing because of this line?

To make sure all the tests are passing I added instructions to README.md at https://github.com/transferwise/pipelinewise-tap-mysql#to-run-tests

Reply:
@koszti hmm sorry about that. I ran the tests locally but for some reason, I thought the one test that failed had nothing to do with my change. Sorry I didn't catch that. It should be fixed now! Thanks!!